spark/src/main/scala/geotrellis/spark/buffer/BufferTilesMethods.scala

```diff
@@ -28,14 +29,15 @@ import scala.reflect.ClassTag
 
 class BufferTilesMethods[
   K: SpatialComponent: ClassTag,
-  V <: CellGrid: Stitcher: ClassTag: (? => CropMethods[V])
+  V <: CellGrid: Stitcher: ClassTag: (? => CropMethods[V]): (? => TilePrototypeMethods[V])
 ](val self: RDD[(K, V)]) extends MethodExtensions[RDD[(K, V)]] {
   def bufferTiles(bufferSize: Int): RDD[(K, BufferedTile[V])] =
     BufferTiles(self, bufferSize)
 
   def bufferTiles(bufferSize: Int, layerBounds: GridBounds): RDD[(K, BufferedTile[V])] =
     BufferTiles(self, bufferSize, layerBounds)
 
+  @deprecated("Please specify buffer sizes per key as a function K => BufferSizes", "1.2")
```

Comment:
Do you think similar function for `Seq` should also be deprecated? (Not rhetorical)